infrastructure/aat.tfvars

```diff
@@ -0,0 +1 @@
+database_backup_retention_days=90
```

Comment:
The retention period can only be between 7 and 35 days.  Set it to 7 unless there's a good reason to keep more days worth of backups in this environment.

Same change needs to be applied to the demo.tfvars, ithc.tfvars and perftest.tfvars files.